index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="AppBar">
+        <header class="ToolBar">
+          <nav>
+            <img src="images/NormalButton.png" alt="" class="NormalButton" />
+          </nav>
+          <div class="Logo">
+            <a href="#">faishar faike</a>
+          </div>
+        </header>
```

Comment:
It's better to align this in the middle and also maybe decrease the height a bit to follow the Figma design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/192716203-527a72db-35f4-43a5-b4d7-bfe91b720831.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/192716261-334e23f1-4bae-4367-96de-c0b831c4136b.png)